fix(coprocessor): fhevm-listener, infinite retry was disabled

rudy-6-4 opened the pull request:
at start the numbre of rety is limited to show any configutaiton issue in operation retry on conncetion loss is infinite, this was broken and limited to 20 retry

jatZama commented:
lgtm once the CI tests end;`

jatZama reviewed (commented):
lgtm once all tests in CI pass.

rudy-6-4 committed:
fix(coprocessor): fhevm-listener, infinite retry was disabled

at start the numbre of rety is limited to show any configutaiton issue
in operation retry on conncetion loss is infinite, this was broken and limited to 20 retry
